modules/reactor/test/tests/meshgenerators/patterned_hex_mesh_generator/positions_meta.data

```diff
@@ -1 +1 @@
-52.1836 -30.1282 0.0
+43.3013 -25 0.0
```

Comment:
The previous code use the average of all external boundary nodes as the origin position, which works well for symmetric reactor cores. Unfortunately this simple example mesh is not symmetric so that the origin is not the mesh centroid (it does not cause any other issue though). With the new mesh centroid calculator, this is fixed.